.codefresh/deploy-build-image.yaml

```diff
@@ -102,4 +102,9 @@ steps:
         candidate: "${{b8_0}}"
         type: push
         registry: atk4
-        tag: "8.0"
+        tag: "8.0""
+      platest:
```

Comment:
latest or platest?

Reply:
it is name of the stage, so I think this is fine